Source/RapyutaSimulationPlugins/Public/Core/RRGameMode.h

```diff
@@ -31,7 +31,7 @@ enum class ERRSimType : uint8
 
 /**
  * @brief GameMode with specific setting, asset loading and ROS2 interface via #ClockPublisher and #ASimulationState.
- * @sa [GameMode and GameState](https://docs.unrealengine.com/4.27/en-US/InteractiveExperiences/Framework/GameMode/
+ * @sa [GameMode and GameState](https://docs.unrealengine.com/5.1/en-US/InteractiveExperiences/Framework/GameMode/
```

Comment:
link arrives on page 404. 
4.27 link is valid

Reply:
Yes, effectively, good to have checked that.